watertap/flowsheets/crystallization/Crystallizer_MVR.py

```diff
@@ -0,0 +1,391 @@
+#################################################################################
+# WaterTAP Copyright (c) 2020-2024, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National Laboratory,
+# National Renewable Energy Laboratory, and National Energy Technology
+# Laboratory (subject to receipt of any required approvals from the U.S. Dept.
+# of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#################################################################################
+from pyomo.environ import (
+    ConcreteModel,
+    TerminationCondition,
+)
+from pyomo.environ import (
+    ConcreteModel,
+    value,
+    Constraint,
+    Objective,
+    Var,
+    NonNegativeReals,
+    TransformationFactory,
+    units as pyunits,
+    check_optimal_termination,
+)
+from pyomo.network import Arc
+from idaes.core import FlowsheetBlock
+from watertap.core.solvers import get_solver
+from idaes.core.util.model_statistics import degrees_of_freedom
+from idaes.core.util.initialization import (
+    propagate_state,
+)
+from pyomo.util.check_units import assert_units_consistent
+
+from idaes.core import FlowsheetBlock
+from idaes.core.util.model_statistics import degrees_of_freedom
+
+import idaes.core.util.scaling as iscale
+import idaes.logger as idaeslog
+from watertap.core.solvers import get_solver
+from idaes.core import UnitModelCostingBlock
+
+from watertap.property_models.unit_specific import cryst_prop_pack as props
+from watertap.unit_models.Crystallizer_revised import Crystallization
```

Comment:
I am interested to see how the revised form differs from the original. For one, I would prefer to have the revised version condensed within the original, accessible through config options, assuming there is considerable overlap in code. Waiting to see this unit model file to determine that.

Reply:
I suppose you can get rid of Crystallizer_revised and just use the revised original now, right?